tools/test-case-generators/format-request-map.json

```diff
@@ -810,6 +841,81 @@
           }
         }
       },
+      "fedora-36": {
```

Comment:
I wonder whether this is needed (and a separate section for each Fedora version).  Was the test case generation failing without this?

Reply:
Yes, it was failing with this error:
`The following custom mountpoints are not supported [“/opt”]`

Reply:
this should not be an issue any more -> https://github.com/osbuild/osbuild-composer/blob/809afbd0ad0d33aaf3f6998ee46e49af22582b83/internal/distro/fedora/distro.go#L49

Reply:
Cool, I just deleted it and updated the manifests